modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/OneOfImplementorAdditionalData.java

```diff
@@ -0,0 +1,137 @@
+package org.openapitools.codegen.utils;
+
+import org.openapitools.codegen.CodegenConfig;
+import org.openapitools.codegen.CodegenModel;
+import org.openapitools.codegen.CodegenProperty;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class holds data to add to `oneOf` members. Let's consider this example:
```

Comment:
This is probably the best javadoc in the project now, and I hope it sets the bar high for everyone :)

Reply:
Thanks :)